docs/docs/responses.mdx

```diff
@@ -2,73 +2,65 @@
 id: responses
 sidebar_label: Responses
 title: Responses
-abstract: Responses are messages that your assistant can send to the user. You can also use responses as actions,
-  making them the simplest type of action.
+abstract: Responses are messages that your assistant sends to the user. A response
+ is typically a text, but it can include other content like images and buttons.
 ---
 ## Defining Responses
 
-A response can be as simple as pure text. Responses can also include random variation, visual
-and interactive components, and more.
+Responses go under the `responses` key in your [domain](./domain.mdx) file. Start each
+response name with `utter_`.
 
-Responses go under the `responses` key in your domain file. Each response name should start with `utter_`.
-This response name is the key you should list your response under.
+Here is an example of a domain file with two responses called `utter_greet` and `utter_bye`:
 
-For example, here is a minimal text response called `utter_greet`:
+```yaml-rasa {4-8} title="domain.yml"
+intents:
+  - greet
 
-```yaml-rasa title="domain.yml"
 responses:
   utter_greet:
   - text: "Hi there!"
+  utter_bye:
+  - text: "See you!"
 ```
 
-### Defining Responses for Retrieval Intents
-
-If you are using retrieval intents in your assistant, you also need to add responses
-for your assistant's replies to these intents:
+If you use [retrieval actions](./retrieval-actions.mdx), make sure you define the responses 
+for each retrieval action in your domain file, like in the following example:
 
 ```yaml-rasa
 responses:
   utter_chitchat/ask_name:
-  - image: "https://i.imgur.com/zTvA58i.jpeg"
-    text: hello, my name is retrieval bot.
   - text: Oh yeah, I am called the retrieval bot.
 
   utter_chitchat/ask_weather:
   - text: Oh, it does look sunny right now in Berlin.
-    image: "https://i.imgur.com/vwv7aHN.png"
-  - text: I am not sure of the whole week but I can see the sun is out today.
 ```
 
-All such responses (e.g. `utter_chitchat/ask_name`) start with the
-`utter_` prefix followed by the retrieval intent name (`chitchat`)
-and the associated response key (`ask_name`).
-
-### Using Variables
+:::note
+Notice the special format of each retrieval action's name. The name starts with `utter_`, 
+followed by the retrieval intent's name (here `chitchat`) and finally a suffix specifying 
+the different response keys (here `ask_name` and `ask_weather`). See 
+[Retrieval Actions](./retrieval-actions.mdx) and examples of 
+[retrieval intents](./training-data-format.mdx#training-examples) to learn more.
```

Comment:
```suggestion
[Retrieval Intents](./training-data-format.mdx#training-examples) to learn more.
```

Reply:
Well, there is no page/section/subsection called Retrieval Intents (maybe there should be!), this is why I left it uncapitalised. @erohmensing should I still go ahead and make this capitalised for the sake of consistency?

Reply:
Good point, keep it as is with now. I'll figure out what the standard on this should be and add it to our styleguide :)